Code/UnitTests/FoundationTest/Basics/Types/IdTest.cpp

```diff
@@ -2,8 +2,12 @@
 
 #include <Foundation/Types/Id.h>
 
-#define EZ_MSVC_WARNING_NUMBER 4463
-#include <Foundation/Basics/Compiler/MSVC/DisableWarning_MSVC.h>
+EZ_WARNING_PUSH()
```

Comment:
what about the matching `EZ_WARNING_POP()` ?

Reply:
There was already one at the end of the file. You just don't see it in the diff.